From da24ffeda9d924e330e8e9bbde4138bb477dcbed Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Thu, 23 Aug 2018 10:09:02 +0200
Subject: [PATCH 65/67] Gallery: use platform cert

This is needed to be able to put Gallery into it's own selinux domain.

Change-Id: I4bf198d20bb2a341e459553d2b519b426fb5d538
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 88fe6756a..7b49d6334 100644
--- a/Android.mk
+++ b/Android.mk
@@ -29,8 +29,8 @@ LOCAL_RESOURCE_DIR += $(LOCAL_PATH)/res
 LOCAL_USE_AAPT2 := true
 
 LOCAL_PACKAGE_NAME := Gallery2
-
 LOCAL_PRIVILEGED_MODULE := true
+LOCAL_CERTIFICATE := platform
 
 LOCAL_PRIVATE_PLATFORM_APIS := true
 
-- 
2.17.1

